src/features/wallet/Wallet.tsx

```diff
@@ -117,6 +126,13 @@ const Wallet: FC<Props> = () => {
             {isConnected ? (
               <Connected>
                 Connected with {onboard?.getState().wallet.name}
+                {/* Testing helper */}
+                {/* <ERC20TransferButton
```

Comment:
this is actually kind of cool, i wonder if this should be a feature to send tokens once you deploy your 2key